Puppetfile.pinned

```diff
@@ -348,7 +348,7 @@ mod 'simp-simpkv',
 
 mod 'simp-simplib',
   :git => 'https://github.com/simp/pupmod-simp-simplib',
-  :tag => '4.2.0'
+  :branch => 'master'
```

Comment:
Are we unpinning all these branches (instead of pinning them to new versions) because we still expect a lot of flux?

Reply:
Yeah, this seems to be the focus of most changes right now.